requirements/build.txt

```diff
@@ -9,5 +9,5 @@ pythran
 numpy>=1.21.1
 
 # Developer UI
-git+https://github.com/scientific-python/devpy
+git+https://github.com/scientific-python/devpy@v0.1
```

Comment:
you have a conflict on pypi. is it really worht sticking to the devpy name?
https://pypi.org/project/devpy/

what name do you want to release on pypi? maybe we can use that instead and move it to conda-forge too.

Reply:
We have received communication from the `devpy` owner that they're willing to let us have it. We'll have to see if we can get that through soon, otherwise I agree with you (and @jarrodmillman who suggested this as well) that we should take a new name and move on.

Unfortunately, any name close to `devpy` is bound to be blocked by `pypi` for being too similar.

Reply:
keep me updated on the choice of name.

devpy doesn't exist on conda-forge, so once the pypi stuff updated, i can release on conda-forge.

https://github.com/conda-forge/devpy-feedstock

Reply:
@gtca and @LucaMarconato  have kindly given us permission to use https://pypi.org/project/spin/ .